client/src/app/shared/components/data-table/data-table.component.html

```diff
@@ -21,12 +21,12 @@
     <ng-container *ngFor="let column of getAllColumns(rows)">
       <ngx-datatable-column prop="{{column}}">
         <ng-template ngx-datatable-cell-template let-rowIndex="rowIndex" let-value="value" let-row="row">
-          <span *ngIf="!isUrl(value)" title="{{value}}">{{value}}</span>
-          <span *ngIf="isUrl(value)">
-          <a target="_blank" href="{{value}}">
-            <mat-icon matTooltip="See Ingest API JSON" [ngStyle]="{'color': 'lightgrey'}">link</mat-icon>
-          </a>
-        </span>
+          <span *ngIf="isUrl(value); else showAsText">
+            <a target="_blank" href="{{value}}">
+              <mat-icon matTooltip="See Ingest API JSON" [ngStyle]="{'color': 'lightgrey'}">link</mat-icon>
```

Comment:
You don't need to use ngStyle here. Just use normal inline `style="color: lightgrey"`. `ngStyle` is for conditional styling

Reply:
This was in the original code. I just copied it over. I have not touched it, but I can do that change.

Reply:
Fixed as requested ;-)